pages/workHeader.xml

```diff
@@ -12,450 +12,458 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                     >123</citedRange></bibl>
+                  <bibl type="CANT"><ptr target="bm:CANT1"/><citedRange unit="item"
+                     >123</citedRange></bibl>
+                  <bibl type="CAVT"><ptr target="bm:CAVT2"/><citedRange unit="item"
+                     >123</citedRange></bibl>
+                  <bibl type="BHL"><ptr target="bm:BHL"/><citedRange unit="item"
+                     >123</citedRange></bibl>
+                  <bibl type="syriaca"><ptr target="bm:syriaca"/><citedRange unit="item"
+                        >123</citedRange></bibl>
+                  <bibl type="KRZ"><ptr target="bm:KRZ"/><citedRange unit="item"
+                     >123</citedRange></bibl>
+               </listBibl>
+            </egXML>
+            <p>You cannot have both <gi>p</gi> and <gi>listBibl</gi> in <gi>sourceDesc</gi>, so if
+               you also need paragraphs, add another <gi>sourceDesc</gi>.</p>
+            <p>Please use only the clavis IDs given in this example. If there is another clavis you
+               would like to add to this list, please open an issue to discuss it. Other references
+               to works in standardized lists, such as Chaîne's Répertoire or various publications
+               listing the single Miracles of Mary can be given as a bibliographic reference in
+                  <gi>listBibl</gi>, while references to external sources such as the <ref
+                  target="https://corpuscoranicum.de/kontexte/uebersicht">Umwelttexte by Corpus
+                  Coranicum</ref> can be given in a <gi>relation</gi> between the record and the
+               external reference in <gi>listRelation</gi>, see below.</p>
+            <p>If there is a need to assign an external clavis ID to textparts of a work, this
+               should be done within <tag>div type="edition"</tag>. A Clavis ID should be assigned
+               for each textpart inside <gi>listBible</gi>:</p>
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+               <div type="textpart" xml:id="Theophilos2">
+                  <label>Extract from the 21st Festal letter (Graf 1937, 383)</label>
+                  <listBibl type="clavis">
+                     <bibl type="CPG">
+                        <ptr target="bm:CPG2"/>
+                        <citedRange unit="item">2589</citedRange>
+                     </bibl>
+                  </listBibl>
+               </div>
+            </egXML>
+         </div>
+
+         <div type="level2">
+            <head>Witnesses</head>
+
+            <p>As stated in <ref target="https://github.com/BetaMasaheft/Documentation/issues/714"
+                  >issue 714</ref>, we always use <gi>witness</gi> for those manuscripts which are
+               used for the edition we are going to (or could potentially) provide in the <tag>div
+                  type='edition'</tag>, if one click 'text' in the upper menu. All manuscript
```

Comment:
I think I would delete "if one click 'text' in the upper menu", because this is secondary.